docs/README.md

```diff
@@ -0,0 +1,107 @@
+| [Tested](#tested)               | 🌑         | 🌑   | 🌑   |
+| [Friendly](#friendly)           | 🌑         | 🌑   | 🌑   |
+| [Prescriptive](#prescriptive)   | 🌑         | 🌑   | 🌑   |
+| [Supported](#supported)         | 🌑         | 🌑   | 🌑   |
+| [Maintained](#maintained)       | 🌑         | 🌑   | 🌑   |
+| [Collaborative](#collaborative) | 🌑         | 🌑   | 🌑   |
+
+Key:
+
+🌑 Not started — 🌔 In progress —  ✅ Meets requirements
+
+## Complete
+
+The SDK includes support for all platform features:
+
+- REST - by offering a generic HTTP request library and a wrapper method for each endpoint
+- GraphQL - by offering a generic GraphQL client that is pre-configured and optimized for GitHub
+- GitHub Apps - by implementing all REST endpoints for GitHub Apps as well as JWT authentication
+- OAuth - by offering middleware in the most popular web framework for performing the OAuth exchange
+- Webhooks - by offering middleware in the most popular web framework for validating webhooks
+
+## Decomposable
+
+Each feature is available both as individual packages and one monolithic meta-package that includes all components pre-wired.
+
+## Generated
+
+Any part of the SDK that varies based on the target GitHub instance should be generated from a schema.
+
+This currently applies to the REST API, webhooks, and any GraphQL wrappers.
+
+## Native
+
+The SDK feels native to the language or platform it is designed for. This includes using native idioms, and following patterns used in other popular SDK libraries for that language.
+
+## Documented
+
+The SDK is uniformly documented, including generated API reference documentation, and getting started guides for using every platform feature.
+
+## Tested
+
+The SDK is well-tested, including thorough unit tests with 100% code coverage and a shared integration test suite that includes VCR-style fixtures.
+
+It is easy to write well-tested applications that use the SDKs.
+
+## Friendly
+
+The SDK anticipates most common issues, gives friendly errors, and points users to help documentation when they run into issues.
+
+## Prescriptive
+
+The SDK follows best practices, such as handling throttling and rate limiting. Where possible, the SDK transparently implements these best practices.
+
+## Supported
+
+New features are supported in a stable release of the SDK at the time of the feature being released.
+
+Developers will receive code-level technical support on client libraries from GitHub.
+
+## Maintained
+
+The SDK is open source, has a community maintainer on contract, and a GitHub employee to oversee the work.
+
+GitHub will:
+
+- Provide advance notice to maintainers for all platform improvements and features
+- Invite maintainers to any relevant events hosted by GitHub.
+
+The community maintainer is expected to:
+
+- Dedicate at least 20 hrs/week (not to exceed 40 hrs/week) to maintaining the client
+- Acknowledge all opened Issues and Pull Requests within 72 hours
```

Comment:
... pull up the comment from below where your GitHub buddy will help you out with coverage. Basically we need a maintainer to be able to take some vacation away from maintenance :)